src/cobra/test/test_core/test_core_reaction.py

```diff
@@ -501,22 +533,23 @@ def test_change_bounds(model):
     assert reac.upper_bound == 2
 
 
-def test_make_irreversible(model):
+def test_make_irreversible(model: Model) -> None:
+    """Test reaction irreversibility."""
     acald_reaction = model.reactions.ACALD
     assert acald_reaction.lower_bound == -1000.0
     assert acald_reaction.upper_bound == 1000.0
     assert acald_reaction.forward_variable.lb == 0.0
     assert acald_reaction.forward_variable.ub == 1000.0
     assert acald_reaction.reverse_variable.lb == 0
     assert acald_reaction.reverse_variable.ub == 1000.0
-    acald_reaction.lower_bound = 0
+    acald_reaction.bounds = (0, acald_reaction.upper_bound)
```

Comment:
We don't need to use `bounds` everywhere. I think it's fine to keep it as it was.